File: Java/Film.java
// Class Film untuk menyimpan data film di bioskop
public class Film {
    private int id;
    private String judul;
    private String genre;
    private int durasi;
    private String gambar;

    // Constructor kosong
    public Film() {
        this.id = 0;
        this.judul = "";
        this.genre = "";
        this.durasi = 0;
        this.gambar = "";
    }

    // Constructor berparameter
    public Film(int id, String judul, String genre, int durasi, String gambar) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.durasi = durasi;
        this.gambar = gambar;
    }

    // Getter dan Setter
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getJudul() {
        return judul;
    }

    public void setJudul(String judul) {
        this.judul = judul;
    }

    public String getGenre() {
        return genre;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }

    public int getDurasi() {
        return durasi;
    }

    public void setDurasi(int durasi) {
        this.durasi = durasi;
    }

    public String getGambar() {
        return gambar;
    }

    public void setGambar(String gambar) {
        this.gambar = gambar;
    }

    // Method untuk menampilkan data
    public void tampilkanData() {
        System.out.println("ID       : " + getId());
        System.out.println("Judul    : " + getJudul());
        System.out.println("Genre    : " + getGenre());
        System.out.println("Durasi   : " + getDurasi() + " menit");
        System.out.println("Gambar   : " + getGambar());
    }
}

File: Python/Film.py
# Class Film untuk menyimpan data film di bioskop
class Film:
    def __init__(self, id: int = 0, judul: str = "", genre: str = "", durasi: int = 0, gambar: str = ""):
        # Atribut private menggunakan __ (encapsulation)
        self.__id = id
        self.__judul = judul
        self.__genre = genre
        self.__durasi = durasi
        self.__gambar = gambar

    # Getter dan Setter
    def getId(self) -> int:
        return self.__id

    def setId(self, id: int) -> None:
        self.__id = id

    def getJudul(self) -> str:
        return self.__judul

    def setJudul(self, judul: str) -> None:
        self.__judul = judul

    def getGenre(self) -> str:
        return self.__genre

    def setGenre(self, genre: str) -> None:
        self.__genre = genre

    def getDurasi(self) -> int:
        return self.__durasi

    def setDurasi(self, durasi: int) -> None:
        self.__durasi = durasi

    def getGambar(self) -> str:
        return self.__gambar

    def setGambar(self, gambar: str) -> None:
        self.__gambar = gambar

    # Method untuk menampilkan data film
    def tampilkanData(self) -> None:
        print(f"ID       : {self.getId()}")
        print(f"Judul    : {self.getJudul()}")
        print(f"Genre    : {self.getGenre()}")
        print(f"Durasi   : {self.getDurasi()} menit")
        print(f"Gambar   : {self.getGambar()}")
